### PR TITLE
ATC flightplans and others

### PR DESCRIPTION
**Pull request by eastwire**
Added:
ATC flightplan packets
xPilot client ID
New VATSIM join message
Kick user packet
Some small typo fixes

### CODE REVIEW

File: docs/api/info.md [hunk 1]
```diff
@@ -284,21 +284,71 @@ $FP(callsign):SERVER:(VFR/IFR):(acft):(spd):(origin):(sched dep):1000:(crz alt):
 VATSIM's flightplan format is simpler, and is based on the [FAA domestic flight plan [PDF]](https://www.faa.gov/documentLibrary/media/Form/FAA_Form_7233-1_7_31_17.pdf).
 
 ```
-$FP(callsign):*A:(VFR/IFR):(acft type):(spd):(origin):(sched dep):(sched dep):(alt):(dest):(EET):(FOB):(alt airport):(remarks)
+$FP(callsign):*A:(VFR/IFR):(acft type):(spd):(origin):(sched dep):(sched dep):(alt):(dest):(EET):(FOB):(alt airport):(remarks):(route)
```
Comment (norrisng) [inline docs/api/info.md:287]: Is `(route)` just the filed route? e.g. `SID WAYPT A123 WAYPT STAR`

Comment (renorris) [reply docs/api/info.md]: Yes

Comment (eastwire) [reply docs/api/info.md:287]: Yes

**Comment by eastwire** [comment]
I've seen this packet: `#PCserver:(atc callsign):CCP:BC:(aircraft callsign):(squawk code)` sent after the main `$FP` packet as a response to a flightplan request: `$CQ(atc callsign):SERVER:FP:(aircraft callsign)`

**Comment by ghost** [comment]
> I've seen this packet: `#PCserver:(atc callsign):CCP:BC:(aircraft callsign):(squawk code)` sent after the main `$FP` packet as a response to a flightplan request: `$CQ(atc callsign):SERVER:FP:(aircraft callsign)`
According to my observation, `#PC` is send for the controller when he / she log in at first as an initialization information of each pilot.

**Pull request closed without merging**